Radzen.Blazor/Rendering/YearView.razor

```diff
@@ -59,8 +59,8 @@
                                 var excessCount = appointments.Count();
                                 string classname = $"rz-slot-title {(day.Month != offSetMonth ? "rz-other-month" : "")} {(excessCount > 0 ? "rz-has-appointments" : "")} {(day == CurrentDate && month == CurrentMonth ? " rz-state-focused" : "")}".Trim();
 
-                                    <div @onclick=@(args => OnListClick(day, appointments)) @attributes=@Attributes(day, "rz-slot")>
-                                        <div class="@classname">
+                                    <div class="rz-slot" @onclick=@(args => OnListClick(day, appointments))>
```

Comment:
This looks as a breaking change as it changes the location where Attributes are applied.